From 42cd1a2d5914ef2904dcccac58a0594027e3058c Mon Sep 17 00:00:00 2001
From: Grzegorz Jaszczyk <jaz@semihalf.com>
Date: Wed, 5 Sep 2018 11:23:42 +0200
Subject: [PATCH 0687/1051] arm64: dts: marvell: armada-cp110: name comphy reg
 range

The reg name will be used by the comphy driver after its rework in
upcoming patch.

Change-Id: I8e015af5a14b07ea119f0d13114be5d9e053fbe5
Signed-off-by: Grzegorz Jaszczyk <jaz@semihalf.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 Documentation/devicetree/bindings/phy/phy-mvebu-comphy.txt | 2 ++
 arch/arm64/boot/dts/marvell/armada-cp110.dtsi              | 1 +
 2 files changed, 3 insertions(+)

diff --git a/Documentation/devicetree/bindings/phy/phy-mvebu-comphy.txt b/Documentation/devicetree/bindings/phy/phy-mvebu-comphy.txt
index bfcf80341657..c07a75ffd686 100644
--- a/Documentation/devicetree/bindings/phy/phy-mvebu-comphy.txt
+++ b/Documentation/devicetree/bindings/phy/phy-mvebu-comphy.txt
@@ -9,6 +9,7 @@ Required properties:
 
 - compatible: should be "marvell,comphy-cp110"
 - reg: should contain the comphy register location and length.
+- reg-names: must contain names corresponding to the comphy register ranges.
 - marvell,system-controller: should contain a phandle to the
                              system controller node.
 - #address-cells: should be 1.
@@ -27,6 +28,7 @@ Example:
 	cpm_comphy: phy@120000 {
 		compatible = "marvell,comphy-cp110";
 		reg = <0x120000 0x6000>;
+		reg-names = "comphy";
 		marvell,system-controller = <&cpm_syscon0>;
 		#address-cells = <1>;
 		#size-cells = <0>;
diff --git a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
index 9b9e61ee85a8..c600d528f340 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
@@ -112,6 +112,7 @@
 		CP110_LABEL(comphy): phy@120000 {
 			compatible = "marvell,comphy-cp110";
 			reg = <0x120000 0x6000>;
+			reg-names = "comphy";
 			marvell,system-controller = <&CP110_LABEL(syscon0)>;
 			#address-cells = <1>;
 			#size-cells = <0>;
-- 
2.17.1

